Source/Core/Common/JsonUtil.h

```diff
@@ -3,10 +3,10 @@
 
 #pragma once
 
-#include <span>
```

Comment:
This is used for the `std::size()`, I think.

Reply:
Originally tried to do the `ToJsonArray` with a span but couldn't make it work.  When I switched to the template function, I forgot to remove that include.  I do need something though, I'll add `cstddef`